acceptance/acceptance_test.go

```diff
@@ -834,30 +833,55 @@ func getNodeTypeID(cloudEnv string) string {
 
 // buildDatabricksBundlesWheel builds the databricks-bundles wheel and returns the path to the wheel.
 // It's used to cache the wheel build between acceptance tests, because one build takes ~10 seconds.
-func buildDatabricksBundlesWheel(t *testing.T, buildDir string) (string, error) {
+func buildDatabricksBundlesWheel(t *testing.T, buildDir string) string {
+	// Clean up directory, remove all but the latest wheel
+	_ = findWheel(t, buildDir)
```

Comment:
Why do this twice? The build will create a new one anyway, so the second call will clear out the old stuff.

Reply:
I don't want to make an assumption that build command always updates mtime.

I also don't want to delete the one wheel that is most likely to be reused. So I'm keeping last mtime wheel there.

Reply:
Added a comment with an explanation. I believe this approach makes least assumptions and so is most resilient.

Reply:
BTW, I checked 'uv build' does not update mtime if up to date whl is already there.